static/app/components/modals/createNewIntegrationModal.tsx

```diff
@@ -0,0 +1,101 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import RadioGroup from 'sentry/components/forms/controls/radioGroup';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+type Props = ModalRenderProps & {
+  orgId: string;
+};
+
+function CreateNewIntegration({Body, Header, Footer, closeModal, orgId}: Props) {
+  const [option, selectOption] = useState('public');
```

Comment:
@NisanthanNanthakumar can you change the default to internal? 99% of the time we want folks to make internal integrations but we still see too many folks creating public ones which might be partly attributed to it being on top.